app/serializers/api/v1/npq_application_serializer.rb

```diff
@@ -63,6 +63,7 @@ class NPQApplicationSerializer
       end
 
       attribute(:eligible_for_funding, &:eligible_for_dfe_funding)
+      attribute(:targeted_delivery_funding_eligibility, &:targeted_delivery_funding_eligibility)
```

Comment:
if it maps the same thing you can do add to line 11 or possibly specify as just `attribute`